tests/unit/crc/main.c

```diff
@@ -187,6 +187,29 @@ ZTEST(crc, test_crc8_ccitt)
 			   sizeof(test2)) == 0xFB, "pass", "fail");
 }
 
+ZTEST(crc, test_crc8_rohc)
+{
+	uint8_t test0[] = { 0 };
+	uint8_t test1[] = { 'A' };
+	uint8_t test2[] = { '1', '2', '3', '4', '5', '6', '7', '8', '9' };
+	uint8_t test3[] = { 0x07, 0x3F };        /* GSM 07.10 example */
+	uint8_t test4[] = { 0x07, 0x3F, 0x89 };  /* GSM 07.10 example */
+	uint8_t test5[] = { 0x03, 0x3f, 0x01, 0x1c };  /* Our GSM 07.10 calc */
+
+	zassert(crc8_rohc(CRC8_ROHC_INITIAL_VALUE, test0,
+			  sizeof(test0)) == 0xcf, "pass", "fail");
```

Comment:
Please use zassert_equal() and ommit the "pass" and "fail" :) Those are implied, and zasser_equal already expands to a descriptive error by default :)

Reply:
Got it, I may wait for all CI passed 😃.
Do I need to correct this one ZTEST(crc, test_crc8_ccitt) as well?

Reply:
Already address it, also with ZTEST(crc, test_crc8_ccitt) as well.